### ISSUE TITLE
Bikeshed update fails

### ISSUE THREAD

**Issue by CxRes** [speced/bikeshed#2946]
Reporting as requested: https://github.com/solid/solid-prep/actions/runs/11762046460/job/32764503788?pr=15#step:3:823

```sh
bikeshed update
    Bringing data files up-to-date...
    Updating via manifest...
    Gathering local manifest data...
    Fetching remote manifest data...
    FATAL ERROR: Something's gone wrong with the remote data; I can't read its timestamp. Please report this!
     ✘  Did not generate, due to errors exceeding the allowed error level.
  Command `bikeshed update` failed with exit code: 2.
  Error: Process completed with exit code 2.
```

**Comment by tabatkins** [comment]
You are stuck on an old Bikeshed, because your CI is configured to use an old Ubuntu which only has Python 3.8. (Bikeshed bumped its minimum Python version to 3.9 some time ago. Unfortunately, I didn't realize this means that `pip install bikeshed` will just silently install the last 3.7-supporting Bikeshed version; had I realized this, I would have made an informative error message...)

The fix is to [update your CI job's Ubuntu version](https://github.com/solid/solid-prep/blob/main/.github/workflows/auto-publish.yml#L11) to one that starts with Python 3.9 or higher; I recommend `ubuntu-latest`.

**Comment by CxRes** [comment]
Your fix works, but I am not sure if I follow. My Windows box has Python 3.9.2 and I still get the same error when running bikeshed.

**Comment by CxRes** [comment]
Uninstalling and reinstalling bikeshed worked. `pip` would not upgrade bikeshed from 3.x to 4.x without that.

**Comment by tabatkins** [comment]
Note: I've now published an old version, advertising 3.7 support, that emits an error message talking about this and asking you to upgrade.
